doc/man1/dgst.pod

```diff
@@ -2,13 +2,14 @@
 
 =head1 NAME
 
-dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5, blake2b, blake2s - message digests
+dgst
+- generate or verify message digests
```

Comment:
Yes, but "generate or verify message *digests*" makes you believe that that you can 'dgst -sha256 file > file.sig' and then verify file.sig. Description paragraph gets it right. Maybe "perform digest operations" would be appropriate?